external-crates/move/crates/move-compiler/tests/move_check/parser/empty_doc_comment.move

```diff
@@ -0,0 +1,16 @@
+module 0x42::m {
+    /* */
```

Comment:
Not sure if there are other cases that need to be tested:
```
// /**/ 
/* **/ 
```

Reply:
Added!